hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -95,22 +96,22 @@ singlePartyHeadFullLifeCycle tracer workDir node@RunningNode{networkId} hydraScr
   refuelIfNeeded tracer node Alice 100_000_000
   -- Start hydra-node on chain tip
   tip <- queryTip networkId nodeSocket
+  let contestationPeriod = UnsafeContestationPeriod 2
```

Comment:
is there a reason why this is 2 and not 1 as the rest?

maybe like in TUI we can create a commented defaultContestationPeriod to re-use.

Reply:
Good catch. The idea of this P.R. is to keep the tests as they were before. So we should not change 1 (old like 103) by 2 here.

```suggestion
  let contestationPeriod = UnsafeContestationPeriod 1
```

Reply:
This is a left-over, I'll bring it back to 1. The changes that actually made this test work are a bit below where I reduced the amount of time we wait from 600 to 10.